libs/traits/src/lib.rs

```diff
@@ -205,7 +205,9 @@ pub trait CurrencyPrice<CurrencyId> {
 
 /// A trait that can be used to calculate interest accrual for debt
 pub trait InterestAccrual<InterestRate, Balance, Adjustment> {
+	type MaxRateCount: Get<u32>;
```

Comment:
It's used for weight calculations for things that consume the InterestAccrual impl.

The other option is to duplicate it as a config option into all pallets that consume InterestAccrual, which feels worse to me

Reply:
I think since this type does not appear in the trait interface, it should be very well documented to teach the user how to use it and why it's needed.